### DISCUSSION TITLE
Unable to generate the projects

### DISCUSSION THREAD

**Discussion by Radhakrishnan-S-12** [spring-io/start.spring.io#1539] [General]
<img width="212" alt="image" src="https://github.com/user-attachments/assets/bf995c69-e41e-4bf2-a97f-c43e2ded14e8">

I'm getting an exception like this. Where I went wrong ?

**Comment by wilkinsona** [comment]
That looks like a bug in [Spring Tools 4](https://github.com/spring-projects/sts4/). Please report it to them. Also, unless the error message is misleading, please note that "Import Getting Started Content" is unrelated to https://start.spring.io. It's a [separate feature](https://github.com/spring-projects/sts4/wiki/Import-A-Spring-Boot-Guide#import-spring-guides-using-the-wizard) that imports guides and tutorials from https://spring.io.

**Reply by Radhakrishnan-S-12** [reply]
Yes. Thats why I raised here. Thanks for the clarification, ill raise there.

**Reply by Radhakrishnan-S-12** [reply]
But even in intelliJ, I'm getting the same.
![image](https://github.com/user-attachments/assets/a5d1ff8b-5d35-4179-9fbf-51d1cf9c0ea9)

**Reply by bclozel** [reply]
can you try generating an application from https://start.spring.io with your browser using the same connection setup (proxy, etc)? The fact that both STS and IntelliJ IDEA fail seem to point that your IP might be flagged by our CDN for security reasons. Maybe completing a simple captcha will unblock you.

**Reply by Radhakrishnan-S-12** [reply]
With start.spring.io im able to create. No issues in that.
